app/model/Raffle.scala

```diff
@@ -50,5 +54,4 @@ case class ReadyToRunRaffle( name:String, prizes:List[String], people:Set[String
 
 }
 
-
-case class FinishedRaffle( name:String, outcome:List[Outcome] ) extends Raffle
\ No newline at end of file
+case class FinishedRaffle( name:String, prizes:List[String], people:Set[String], outcome:List[Outcome] ) extends Raffle
```

Comment:
We were implementing this when security showed up.  We needed somewhere to store the state of an ongoing raffle (during the `run` and `finished` invocations).  We've since added another state `OngoingRaffle`.  We'll remove `prizes` and `people` from the `FinishedRaffle`.

We committed some WIP to allow the other groups to see the API, even though it was still changing.